packages/core/upload/server/routes/admin.js

```diff
@@ -139,5 +139,21 @@ module.exports = {
         ],
       },
     },
+    {
+      method: 'POST',
```

Comment:
Shouldn't this a `GET` route?

Reply:
It's weird I know :p I'm not sure what to do. I would be glad to have alex opinion on this, for future routes too.